model-optimizer/extensions/back/SpecialNodesFinalization.py

```diff
@@ -123,7 +123,8 @@ def pattern():
         return dict(
             nodes=[
                 ('const_data', {'kind': 'data', 'value': lambda value: value is not None}),
-                ('result_node', {'type': 'Result', 'kind': 'op'}),
+                ('result_node', {'type': 'Result', 'kind': 'op',
```

Comment:
Please, take a look at unit tests for this functionality in `model-optimizer/extensions/back/SpecialNodesFinalization_test.py` file and update them with this new attribute (and add negative test cases when value of this attribute is `True`).